plugins/npm/npm.plugin.zsh

```diff
@@ -1,4 +1,12 @@
-eval "$(npm completion 2>/dev/null)"
+local __NPM_COMPLETION_DIR="${0:A:h}"
```

Comment:
Yuo have to use the `local` command in function.
This case, `local` command is not necessary. you can remove it.